### PR TITLE
Ana/display Cosmos topVoters picture when validator

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Mini-PR. Simply display also topVoter picture if it is a validator:

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6a2acca513e000894e4127--lunieio.netlify.app

File: app/src/components/governance/ParticipantList.vue [hunk 1]
```diff
@@ -9,7 +9,13 @@
       >
         <div class="first-column">
           <span class="icon">
-            <img v-if="participant.picture" :src="participant.picture" />
+            <img
```
Comment (faboweb) [inline app/src/components/governance/ParticipantList.vue:12]: wrong fix. the picture is not set in the API

**Review by faboweb — CHANGES_REQUESTED** [review]
wrong fix

File: app/src/components/governance/ParticipantList.vue (hunk 1 above)
Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:12]: OK, let me fix this in API

**Comment by Bitcoinera** [comment]
Seems one of the councellors is a validator :star_struck: :

image

**Review by jbibla — APPROVED** [review]

**Pull request merged**